onnxruntime/core/graph/graph.cc

```diff
@@ -3847,15 +3857,20 @@ void Graph::CancelFuseSubGraph(const Node& fused_node) {
     return;
 
 #if !defined(ORT_MINIMAL_BUILD)
-  // Remove the tempoary schema from schema container container
-  auto* temp_schema_ptr = fused_node.Op();
-  auto it = std::find_if(
-      fused_schemas_containers_.begin(), fused_schemas_containers_.end(),
-      [temp_schema_ptr](const std::unique_ptr<ONNX_NAMESPACE::OpSchema>& schema) {
-        return schema.get() == temp_schema_ptr;
-      });
-  if (it != fused_schemas_containers_.end()) {
-    fused_schemas_containers_.erase(it);
+  // Remove the temporary schema from schema container container
+  const auto* temp_schema_ptr = fused_node.Op();
+  auto schema_key = onnxruntime::MakeString(temp_schema_ptr->domain(), "_",
+                                            temp_schema_ptr->Name());
+  if (fused_schemas_map_.contains(schema_key)) {
```

Comment:
Seems it's inevitably to have  double lookup as we have two containers. because we don't know how the schema created even resuable_fused_schemas_map_ only hold its reference.